ci-rust

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+. ./config
+
+mkdir -p "$TEST_OUT_DIR/rust"
+
+pushd spec/rust
+cargo test-junit \
+      --name "../../$TEST_OUT_DIR/rust/results.xml"
+popd
```

Comment:
You can actually just do `cd spec/rust` and not worry about returning back — this one is always executed in a separate shell, not sourced or anything like that.